db_lesson.sql

```diff
@@ -0,0 +1,815 @@
+mysql> USE db_lesson;
+Database changed
+mysql> SHOW TABLES;
+Empty set (0.00 sec)
+
+mysql> CREATE TABLE people (
+  -> id INT,
+  -> name VARCHAR(255),
+  -> email VARCHAR(255),
+  -> password CHAR(8)
+  -> );
```

Comment:
[must]
.sql拡張子のファイルは、ファイル読み込みで一気にSQLを実行できるようにSQLを書いておく用途に使われます。
SQL文以外の文が書いてあるとうまく読み込めずにエラーになってしまいますので、SQL以外の文字（日本語や実行結果など）はコメントアウトするか無視しましょう！

また、`mysql> `や`  -> `という記述も、コマンド画面上の表示で先頭を表したり改行を表したりする文字であり、SQL文ではありませんので消しましょう！

```suggestion
CREATE TABLE people (
id INT,
name VARCHAR(255),
email VARCHAR(255),
password CHAR(8)
);
```